Gems/ROS2/Code/Source/RobotImporter/URDF/CollidersMaker.cpp

```diff
@@ -387,20 +387,18 @@ namespace ROS2
 
             // Get asset product id (pxmesh)
             AZ::Data::AssetId assetId;
-            AZ::Data::AssetType assetType = AZ::AzTypeInfo<PhysX::Pipeline::MeshAsset>::Uuid();
+            const AZ::Data::AssetType PhysxMeshAssetType = PhysX::Pipeline::MeshAsset::TYPEINFO_Uuid();
```

Comment:
There is a macro helper for this:

````
const AZ::Data::AssetType PhysxMeshAssetType = azrtti_typeid<PhysX::Pipeline::MeshAsset>();
````

Reply:
Also, since it's a local variable it should start lowercase: `PhysxMeshAssetType`  -> `physxMeshAssetType `